src/value.rs

```diff
@@ -427,6 +427,50 @@ impl Value {
     }
 }
 
+/// Typed Simfony value.
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct TypedValue {
+    value: Value,
+    ty: ResolvedType,
+}
+
+impl TypedValue {
+    /// Create a `value` of the given `ty`pe.
```

Comment:
In 03b24212d95fdd60b3e23cbdd510911bcabe2b2d:

typo `type`

Reply:
I see you don't like my attempt at highlighting the parameter `ty` which cannot be called `type` because that would be a keyword :) I admit that "`ty`pe" looks weird and should be just "type".

Reply:
Oh, lol :) yeah, I see what you were trying to do. But sadly it's not obvious to a reader. I think you should just use "type" without any backticks.